handbook/information-security/supplier-policy.md

```diff
@@ -0,0 +1,115 @@
+# Supplier Policy
+
+In providing our services we rely on a number of third party suppliers that have
+access to some of the sensitive information we handle. This policy sets out how
+we vet our suppliers to ensure that we understand and assess the risks involved.
+
+## Objectives
+
+The objectives of our Supplier Policy are to:
+* understand which of our suppliers are handling sensitive information on our
+  behalf
+* understand the risks created by our collaboration
+* ensure the relevant suppliers follow best practices to minimise information
+  security risks
+* work with our suppliers to implement best practice where it is not being
+  implemented
+* make a conscious decision whether the risks we expose ourselves to are
+  acceptable
+* where we are exposed to unacceptable risk, mitigate this by ending the
+  collaboration if necessary
+
+## Vetting Process
+The below diagram illustrates the process that the rest of this document
+describes in more detail.
+
+![Supplier Policy Flowchart](../../assets/supplier-policy.png)
+
+### Supplier List
+
+Our [Supplier
+List](https://docs.google.com/spreadsheets/d/14kog_nFut3v2t5mZNNatZ-oWb7tP3ImWdXjUP9EkZGc/edit?usp=sharing)
+tracks the suppliers we interact with.
+
+This is checked for completeness monthly by the [Information Security
+Officer](../README.md#contacts).
+
+We define suppliers as any organisation that has invoiced us. Therefore the
```

Comment:
I think we need to change this. If we just started working with someone, they may not have invoiced us yet. They may also be providing services in return for our services.

Reply:
How about "We define suppliers as any organisation that we collaborate with in providing our clients with our services"?

Reply:
We define suppliers as any organisation that we collaborate with in providing services to our clients.